packages/api-reference/src/components/ApiReferenceLayout.vue

```diff
@@ -277,20 +270,8 @@ provide(WORKSPACE_SYMBOL, workspaceStore)
 const activeEntitiesStore = createActiveEntitiesStore(workspaceStore)
 provide(ACTIVE_ENTITIES_SYMBOL, activeEntitiesStore)
 
-provide(
-  HIDE_DOWNLOAD_BUTTON_SYMBOL,
-  () => props.configuration.hideDownloadButton,
-)
-provide(
-  HIDE_TEST_REQUEST_BUTTON_SYMBOL,
-  () => props.configuration.hideTestRequestButton,
-)
-provide(OPENAPI_DOCUMENT_URL_SYMBOL, () => props.configuration.spec?.url)
-provide(INTEGRATION_SYMBOL, () =>
-  props.configuration._integration !== null
-    ? props.configuration._integration
-    : 'vue',
-)
+// Provide the configuration
+provide(CONFIGURATION_SYMBOL, props.configuration)
```

Comment:
Provide a single time for the config now

Reply:
Finally!